server/index.js

```diff
@@ -10,85 +10,91 @@ const path = require('path');
 const fs = require('bfile');
 const { execSync } = require('child_process');
 const os = require('os');
-const Config = require('bcfg');
-const logger = require('./logger');
-const Blgr = require('blgr');
+const { createLogger } = require('./logger');
+const Logger = require('blgr');
```

Comment:
we can delete this line, unused import

Reply:
Actually it still is when the main logger is created below, but it doesn't need to be. Now that you've made me look at it, I should be using createLogger. Will update 👍

Reply:
updated